__tests__/app.tests.js

```diff
@@ -49,3 +49,32 @@ describe("GET api ", () => {
       });
   });
 });
+describe("GET api/articles/ID ", () => {
+  test("GET articles by id should return article from ID ", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        // console.log(res.body.Articles[0]);
+        const output = res.body.articles[0];
+        expect(typeof res.body).toEqual("object");
+        expect(output).toHaveProperty("author");
```

Comment:
I would suggest this is insufficient testing, we have not checked the values are of the correct data types, we will also want to check the article id is for the one we are using in the request